category-foreign-agent.php

```diff
@@ -114,7 +122,7 @@ function nm_render_podcast_credit($credit) {
   <section class="container pt-4 pb-4">
     <div class="flex-grid-row mb-4">
       <div class="flex-grid-item flex-item-xxl-12 mb-4">
-        <h4 class="font-size-9 text-uppercase font-weight-bold" style="color: #E3FFFF;"><a href="<?php echo home_url('category/audio/'); ?>">Podcast</a></h4>
+        <a href="<?php echo home_url( 'category/audio/' ); ?>" class="ui-tag-block"><span class="ui-tag">Podcast</span></a>
```

Comment:
Because of its usage here on top of imagery probably should be the varient used on primary featured images with no border and the other hover state. But this is a design detail so perhaps intentional/to be checked by Pietro.

Reply:
Same on the new Committed page

Reply:
Its been design reviewed multiple times but will ask about this specific detail, ok

Reply:
Its been through design review and passed but i will flag this specific bit